src/test/txrebroadcast_tests.cpp

```diff
@@ -96,7 +142,7 @@ BOOST_AUTO_TEST_CASE(max_rebroadcast)
 
     // Check that the transaction gets returned to rebroadcast
     std::vector<TxIds> candidates = tx_rebroadcast.GetRebroadcastTransactions();
-    BOOST_CHECK_EQUAL(candidates.size(), 1);
+    BOOST_REQUIRE_EQUAL(candidates.size(), 1);
```

Comment:
In 0f13d876977d6a83549fa40f68a73308ba10d95d "[test] Add unit test for the fee rate cache"

Seems like this change should be part of the previous commit.

Reply:
fixed